tests/test_raster/test_raster.py

```diff
@@ -1929,20 +1917,14 @@ def test_split_bands(self) -> None:
             red.data.data.squeeze().astype("float32"), img.data.data[0, :, :].astype("float32"), equal_nan=True
         )
 
-        # Modify the red band and make sure it propagates to the original img (it's not a copy)
-        red.data += 1
```

Comment:
Why did you delete this test? Did you mean to replace "red.data += 1" with "red.data = red.data + 1"?

Reply:
Added back too. Old change from September...